esmvalcore/preprocessor/_area.py

```diff
@@ -250,9 +250,22 @@ def area_statistics(cube, operator, fx_variables=None):
     grid_areas = tile_grid_areas(cube, fx_variables)
 
     if not fx_variables and cube.coord('latitude').points.ndim == 2:
-        logger.error(
-            'fx_file needed to calculate grid cell area for irregular grids.')
-        raise iris.exceptions.CoordinateMultiDimError(cube.coord('latitude'))
+        if 'grid_latitude' in [coord.standard_name for coord in cube.coords()]:
+            coord_names = ['grid_longitude', 'grid_latitude']
+            cube = guess_bounds(cube, coord_names)
```

Comment:
if they already have bounds that'll fail

Reply:
Actually no, _shared.guess_bounds checks for that.

Reply:
should have known better, I actually wrote that myself :grin: